src/app/component/eco-news/create-news/create-news.component.css

```diff
@@ -182,27 +189,79 @@ button {
     color: red;
 }
 
-@media screen and (max-width: 1000px) {
+@media screen and (max-width: 1024px) {
```

Comment:
![image](https://user-images.githubusercontent.com/54113846/82150012-3fa0b300-9860-11ea-9ccb-85907bcf3d92.png)
Please, compare it with the mockup (breakpoint 320 px)

Reply:
fixed